go.mod

```diff
@@ -78,4 +79,9 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.21.1
 	k8s.io/sample-controller => k8s.io/sample-controller v0.21.1
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.9.0
+
+	// OCP-specific
+	// This version should be the same as the one set in the Makefile from the upstream tag:
+	// https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.3.0-rc.3/Makefile#L132
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.2.5
```

Comment:
Pinning the controller-tools version so that we use the same version that's hardcoded in the Makefile. This is not required, but not doing this will fetch newer k8s deps, and at this point I think it's better to not differ to much from upstream.